kakao/src/main/kotlin/com/agoda/kakao/picker/KTimePicker.kt

```diff
@@ -0,0 +1,28 @@
+import android.widget.TimePicker
+import androidx.test.espresso.contrib.PickerActions
+import com.agoda.kakao.common.views.KBaseView
+import com.agoda.kakao.text.KButton
+
+/**
+ * View for interact with default time picker
+ *
+ * @see TimePicker
+ */
+class KTimePicker : KBaseView<KTimePicker>({ isInstanceOf(TimePicker::class.java) }) {
```

Comment:
`TimePicker` is a default widget and you should leave the ability of matching it by Kakao users. Please provide all base class constructors without additional logic.

Reply:
We have the same logic for SnackBar

Reply:
Yeah, but you can't use snackbar view directly, only through the `Snackbar.show()`.
Whereas DatePicker and TimePicker are normal widgets and can be included in any layout.